src/style.css

```diff
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir-bold";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: bold;
+  font-weight: 900;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+input::-webkit-search-cancel-button {
+  -webkit-appearance: none;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin-top: 20px;
+}
+
+.form-big {
+  width: 100%;
+  height: 70px;
+}
+
+.form-small {
+  width: 100%;
+  height: 42px;
+}
+
+.search-label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-label::after {
+  position: absolute;
+  content: "";
+  display: block;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.label-big::after {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.label-small::after {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-text {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
```

Comment:
Use `padding` instead `height`

Reply:
I don't understand, what do you meen? 
In my project focus don't have design or at Figma?